server/jamovi/server/compute/functions.py

```diff
@@ -244,6 +244,16 @@ def VMED(values: float):
     return stats.median(values)
 
 
+@column_wise
+def VMAD(values: float):
+    return VMED(map(lambda v: abs(v - VMED(values)), values))
```

Comment:
ah, i think this was better when you assigned `VMED(values)` to an intermediate value. in it's current form, it executes `VMED(values)` once for every value.

Reply:
Thanks for drawing attention to this.
He got out of hand.
Guess I'm taking your time 🤔